cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,219 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-logging"]
+= Tutorial: Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-logging
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-05-01
+
+There are various methods to view your logs in {product-rosa} (ROSA). Use the following procedures to forward the logs to AWS CloudWatch, view the logs directly through the pod by using `oc logs`, and use Kibana (EFK Stack) to search the logs.
```

Comment:
EFK is deprecated. it it better to use vector  lokistack and view logs in Webconsole.

Reply:
There are various methods to view your logs in {product-rosa} (ROSA). Use the following procedures to forward the logs to AWS CloudWatch, view the logs directly through the pod by using `oc logs`, use Kibana (EFK Stack) to search the logs,  or use webconsole pannel observe-logs(lokistack) to search logs.